docs/ci-scripts.md

```diff
@@ -9,22 +9,56 @@ The Automator shell scripts allow your CI platform to start a crawl on a develop
 The following environment variables should be set in your CI platform to allow the scripts to work correctly.
 
 ```
-AUTOMATOR_USER_KEY_ID (Key ID for generated API Key)
+AUTOMATOR_USER_KEY_ID (-userKeyId) (Key ID for generated API Key)
 
-AUTOMATOR_USER_KEY_SECRET (Secret value for generated API Key)
+AUTOMATOR_USER_KEY_SECRET (-userKeySecret) (Secret value for generated API Key)
 
-AUTOMATOR_TEST_SUITE_ID (optional) // can be passed into ci as param
+AUTOMATOR_TEST_SUITE_ID (-testSuiteId) (optional) // can be passed into ci as param
 
-AUTOMATOR_START_ONLY (optional, set to `1` or `true` to run Automator without blocking a build)
+AUTOMATOR_START_ONLY (-startOnly) (optional, set to `1` or `true` to run Automator without blocking a build)
 ```
 
-example:
-**AUTOMATOR_TEST_SUITE_ID** replace with your id
+### PowerShell
+
+Example 1:
+
+```
+$env:AUTOMATOR_USER_KEY_ID = <number>
+$env:AUTOMATOR_USER_KEY_SECRET = <string>
+$env:AUTOMATOR_TEST_SUITE_ID = <string>
+$env:AUTOMATOR_START_ONLY = <boolean> | 1 | 0
+./ci.ps1
+```
+
+Example 2: 
+
+* replace **AUTOMATOR_TEST_SUITE_ID** with your id
```

Comment:
Isn't this optional as mentioned above in line 16

Reply:
Not for example 2.